Modules/posixmodule.c

```diff
@@ -16634,6 +16825,16 @@ static const struct have_function {
     {"HAVE_EVENTFD", NULL},
 #endif
 
+#ifdef HAVE_TIMERFD_CREATE
+    {"HAVE_TIMERFD_CREATE", NULL},
+#endif
+#ifdef HAVE_TIMERFD_SETTIME
+    {"HAVE_TIMERFD_SETTIME", NULL},
+#endif
+#ifdef HAVE_TIMERFD_GETTIME
+    {"HAVE_TIMERFD_GETTIME", NULL},
+#endif
```

Comment:
It seems to me these two defines are not used to guard their respective functions. If you only need to check for `timerfd_create` to detect this feature, you can remove the `timerfd_settime` and `timerfd_gettime` checks from configure.ac; if not, you can add preprocessor guards for those respective functions in posixmodule.c.

Reply:
Alternatively, you could consolidate all of them in a single `HAVE_TIMEFD_API` define (in configure.ac), and simply use that as a guard in posixmodule.c.

Reply:
@erlend-aasland 
Fixed.